sdk/docs/KFServingApi.md

```diff
@@ -0,0 +1,137 @@
+# KFServingApi
+
+Class | Method |  Description
+------------ | ------------- | -------------
+KFServingApi | [deploy](#deploy) | Create KFService|
+KFServingApi | [get](#get)    | Get the specified KFService|
+KFServingApi | [patch](#patch)  | Patch the specified KFService|
+KFServingApi | [delete](#delete) | Delete the specified KFService |
+
+## deploy
+> deploy(kfservice, namespace=None)
+
+Create the provided KFService in the specified namespace
+
+### Example
+
+```python
+from kubernetes import client
+
+from kfserving.api.kf_serving_api import KFServingApi
```

Comment:
This seems a little awkward to me. Is it not possible to do something like `from kfserving import KFServingClient`

Reply:
Thanks @ellis-bigelow Enhanced, now that can be import as `from kfserving import KFServingClient`.